bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,271 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.Diff import Diff
+
+
+def generate_spacing_diff(file, filename, line, line_number,
+                          match_object, required_spacing):
+    '''
+    Generate a diff for incorrectly spaced control or variable tags.
+
+    :param match_object:     the match object generated by a match on the
+                             regular expression control_start_regex,
+                             control_end_regex or variable_regex in the
```

Comment:
As control_end_regex, control_start_regex, variable_regex are variables using backticks for code block here would be easier to understand